projects/ad9361/src/main.c

```diff
@@ -70,14 +73,22 @@
 #include "iio_axi_dac.h"
 #include "iio_ad9361.h"
 #include "uart.h"
+#include "iio_app.h"
+
+#ifdef XILINX_PLATFORM
 #include "uart_extra.h"
 #include "xil_cache.h"
+#endif //XILINX
 
 #endif // IIO_SUPPORT
 
 /******************************************************************************/
 /************************ Variables Definitions *******************************/
 /******************************************************************************/
+
+#ifdef LINUX_PLATFORM
+#endif
+
```

Comment:
Is there a purpose to this?

Reply:
sorry, i was trying something right now and i pushed that by mistake, i'll remove it asap